hack/upgrade-test-harness/testdata/alpha/install.yaml

```diff
@@ -0,0 +1,970 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Have you considered downloading the manifest from https://download.elastic.co/downloads/eck/0.9.0/all-in-one.yaml?

Reply:
Yes. I didn't do so for several reasons:
- We'll lose the installation file if the content is removed from the website or if the URLs are shuffled around
- Downloading over the internet increases the chances of flakiness due to network issues
- If we run the test regularly, our download stats will be artificially inflated

Reply:
> Yes. I didn't do so for several reasons:

That seems reasonable. 
 
> * We'll lose the installation file if the content is removed from the website or if the URLs are shuffled around

I think that if we want to prevent that, this is not the right place to backup the yaml. If the URLS are shuffled around, we will have to update the doc anyway.

> * Downloading over the internet increases the chances of flakiness due to network issues

Yes, but I tend to hope that hitting S3 is very very stable.

> * If we run the test regularly, our download stats will be artificially inflated

Yes, but there is a trick to avoid that :).

Maybe I'm nitpicking but it seems wrong for me. I understand these reasons and I find them relatively reasonable. I will not oppose putting the yaml in git but I admit that it bothers me a little.